ansible/ci-build-collector.yml

```diff
@@ -52,46 +52,60 @@
         username: "{{ stackrox_io_username }}"
         password: "{{ stackrox_io_password }}"
 
-    - name: Push slim to quay.io/stackrox-io
+    - name: Push to quay.io/stackrox-io
       community.docker.docker_image:
-        name: "{{ collector_image }}-{{ arch }}-slim"
+        name: "{{ collector_image }}-{{ arch }}"
         push: true
         source: local
 
     - name: Push base to quay.io/stackrox-io
       community.docker.docker_image:
-        name: "{{ collector_image }}-{{ arch }}-slim"
+        name: "{{ collector_image }}-{{ arch }}"
         repository: "{{ collector_image }}-{{ arch }}-base"
         push: true
         source: local
 
+    - name: Push slim to quay.io/stackrox-io
+      community.docker.docker_image:
+        name: "{{ collector_image }}-{{ arch }}"
+        repository: "{{ collector_image }}-{{ arch }}-slim"
+        push: true
+        source: local
+
     - name: Login to quay.io
       community.docker.docker_login:
         registry_url: quay.io
         username: "{{ rhacs_eng_username }}"
         password: "{{ rhacs_eng_password }}"
 
-    - name: Push slim to quay.io/rhacs-eng
+    - name: Push to quay.io/rhacs-eng
       community.docker.docker_image:
-        name: "{{ collector_image }}-{{ arch }}-slim"
-        repository: "{{ rhacs_eng_image }}-{{ arch }}-slim"
+        name: "{{ collector_image }}-{{ arch }}"
+        repository: "{{ rhacs_eng_image }}-{{ arch }}"
         push: true
         source: local
 
-    - name: Push base to quay.io/stackrox-io
+    - name: Push base to quay.io/rhacs-eng
       community.docker.docker_image:
-        name: "{{ collector_image }}-{{ arch }}-slim"
+        name: "{{ collector_image }}-{{ arch }}"
         repository: "{{ rhacs_eng_image }}-{{ arch }}-base"
         push: true
         source: local
 
+    - name: Push base to quay.io/rhacs-eng
+      community.docker.docker_image:
+        name: "{{ collector_image }}-{{ arch }}"
+        repository: "{{ rhacs_eng_image }}-{{ arch }}-slim"
+        push: true
+        source: local
+
     - name: Print images pushed
       debug:
         msg:
         - "Pushed the following images:"
-        - "  {{ collector_image }}-{{ arch }}-slim"
+        - "  {{ collector_image }}-{{ arch }}"
         - "  {{ collector_image }}-{{ arch }}-base"
-        - "  {{ rhacs_eng_image }}-{{ arch }}-slim"
+        - "  {{ rhacs_eng_image }}-{{ arch }}"
         - "  {{ rhacs_eng_image }}-{{ arch }}-base"
```

Comment:
Instead of removing the -slim tags, we might want to add the -latest and this one to the list of images we push, that way CI can just create all the multiarch manifest in a single job.

```yml
        - "Pushed the following images:"
        - "  {{ collector_image }}-{{ arch }}"
        - "  {{ collector_image }}-{{ arch }}-latest"
        - "  {{ collector_image }}-{{ arch }}-slim"
        - "  {{ collector_image }}-{{ arch }}-base"
        - "  {{ rhacs_eng_image }}-{{ arch }}"
        - "  {{ rhacs_eng_image }}-{{ arch }}-latest"
        - "  {{ rhacs_eng_image }}-{{ arch }}-slim"
        - "  {{ rhacs_eng_image }}-{{ arch }}-base"
```